src/celeritas/em/interactor/detail/PhysicsConstants.hh

```diff
@@ -64,28 +64,22 @@ CELER_CONSTEXPR_FUNCTION units::MevEnergy seltzer_berger_upper_limit()
     return units::MevEnergy{1e3};  //! 1 GeV
 }
 
-//! Energy threshold between muon ionization models. TODO: make configurable
-CELER_CONSTEXPR_FUNCTION units::MevEnergy mu_bethe_bloch_lower_limit()
+//! Maximum energy for the Bragg and ICRU73QO models
+CELER_CONSTEXPR_FUNCTION units::MevEnergy bragg_icru73qo_upper_limit()
```

Comment:
With your refactor, is this *only* now used in the construction of the models by the process? If so that's fantastic: it always bothered me to have a constexpr arbitrary value like this used in several places. Moving these to the process options would allow the values to be configurable as well.

Reply:
Yep, only in the process... I'll move them there.